src/test/java/htsjdk/samtools/util/StringLineReaderTest.java

```diff
@@ -27,6 +27,8 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+/** @deprecated testing a deprecated class */
```

Comment:
I would delete this test instead of deprecating it, as there is no interesting code left to test.

Reply:
True. Removed.